extensions/powershell/src/project.ts

```diff
@@ -76,6 +78,7 @@ export class Project extends codeDomProject {
 
     const mil = await service.GetValue('max-inlined-parameters');
     this.maxInlinedParameters = typeof mil === 'number' ? mil : 4;
+    this.moduleVersion = await service.GetValue('module-version') || '1.0.0';
```

Comment:
Should we force a less-attractive default like `0.1.0` to encourage people to add a `module-version` ?